src/App.tsx

```diff
@@ -22,7 +25,21 @@ const App = (): any => {
   };
 
   const [state, setState] = useState(initialState);
-  
+  // ------------- Firebase setup Tutorial ----------------
+  // const [show, setShow] = useState(false);
+  const [show, setShow] = useState(false);
+  const [notification, setNotification] = useState({title: '', body: ''});
+  const [isTokenFound, setTokenFound] = useState(false);
+  fire.getToken(setTokenFound);
+
+  fire.onMessageListener().then(payload => {
+    setShow(true);
+    setNotification({title: payload.notification.title, body: payload.notification.body})
+    console.log(payload);
+  }).catch(err => console.log('failed: ', err));
+  // ------------------------------------------
+
+
```

Comment:
We should move this to a different component. something like a bell icon (similar to magic bell)

Reply:
Created a component called BellIcon in /Components/Notification directory